numpy/lib/function_base.py

```diff
@@ -1491,20 +1491,30 @@ def gradient(f, *varargs, **kwargs):
     """
     Return the gradient of an N-dimensional array.
 
-    The gradient is computed using second order accurate central differences
-    in the interior and either first differences or second order accurate
-    one-sides (forward or backwards) differences at the boundaries. The
-    returned gradient hence has the same shape as the input array.
+    The gradient (defined as :math:`\\nabla f = [\\frac{\\partial f}{\\partial x},  
```

Comment:
We try to keep the latex bits in the notes because they aren't very readable in the terminal. I don't think the definition is really needed here.